README.md

```diff
@@ -12,7 +12,9 @@
 
 This is a video of an app in development, and it will be updated as needed.
 
-[Screen_recording_20230805_160952.webm](https://github.com/DroidKaigi/conference-app-2023-images/assets/1386930/32968ea9-6ef8-4707-b402-0fd009cc04fe)
+<video src="https://github.com/DroidKaigi/conference-app-2023/assets/13657682/ec91d1b8-639d-4622-8371-1387be4c6bac" width="320" controls>
```

Comment:
Just for your reference, you can use demo mode in OS's developers option to fix the time and status bar

Reply:
I did not know this!
Thanks for letting us know.